test/e2e-common.sh

```diff
@@ -258,6 +258,27 @@ function install_knative_serving_standard() {
   # If we need to build from source, then kick that off first.
   if [[ -z "$1" ]]; then
     build_knative_from_source
+  else
+    SERVING_ISTIO_YAML="${1}"
+  fi
+
+  echo ">> Installing Knative namespace and CRD"
+  echo "Namespace YAML: ${SERVING_NAMESPACE_YAML}"
+  echo "CRD YAML: ${SERVING_CRD_YAML}"
+  kubectl apply \
+	  -f "${SERVING_NAMESPACE_YAML}" \
+	  -f "${SERVING_CRD_YAML}" || return 1
+  UNINSTALL_LIST+=( "${SERVING_NAMESPACE_YAML}" "${SERVING_CRD_YAML}" )
+
+  echo ">> Installing Ingress"
+  if [[ -n "${GLOO_VERSION}" ]]; then
+    install_gloo
+  elif [[ -n "${KOURIER_VERSION}" ]]; then
+    install_kourier
+  elif [[ -n "${AMBASSADOR_VERSION}" ]]; then
+    install_ambassador
+  else
+    install_istio "${SERVING_ISTIO_YAML}"
```

Comment:
Can we use switch instead of if-else ?
and in default case we can keep `install_istio`

Reply:
The readability of bash's `if` statement is not so bad, so I feel that we don't force it to use switch(case). But if we want to change anyway, we should do the same in `test_setup()` and another PR is better.